Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -49,10 +57,22 @@ class SCENE_SURFACE_MESH_ITEM_EXPORT Scene_surface_mesh_item
   Bbox bbox() const Q_DECL_OVERRIDE;
   QString toolTip() const Q_DECL_OVERRIDE;
 
+  // Only needed for Scene_polyhedron_item
+  void setItemIsMulticolor(bool){}
+  void update_vertex_indices(){}
+  void update_halfedge_indices(){}
+  void update_facet_indices(){}
+
+  std::vector<QColor>& color_vector();
+  void set_patch_id(SMesh::Face_index f,int i)const;
+  int patch_id(SMesh::Face_index f)const;
```

Comment:
I have already said in my previous review that the item type should not have any function to get/set the `patch_id` of a face. That is too low-level for the items API: a plugin using a `Surface_mesh` should call `item->face_graph()` to get the pointer to the `Surface_mesh`, and then get/set the patch ids using the CGAL object, and not the item object, that should not know anything about CGAL.